git-russian-roulette

```diff
@@ -6,15 +6,13 @@
 # disable Ctrl+c and Ctrl+z
 trap '' INT TSTP
 
-GIT_DIR=$(git rev-parse --show-toplevel)
```

Comment:
Why did you removed this one?

Next call (#L10) try to check if it's a git directory or not, so `$?` will be the exit code of `trap` placement.